PiwikTracker/PiwikUserDefaults.swift

```diff
@@ -0,0 +1,94 @@
+import Foundation
+
+/// PiwikUserDefaults is a wrapper for the UserDefaults with properties
+/// mapping onto values stored in the UserDefaults.
+/// All getter and setter are sideeffect free and automatically syncronize
+/// after writing.
+internal class PiwikUserDefaults {
+    internal static let standard = PiwikUserDefaults()
+    internal let userDefaults = UserDefaults.standard
+    
+    internal var totalNumberOfVisits: UInt32 {
+        get {
+            return UInt32(self.userDefaults.integer(forKey: PiwikUserDefaults.Constants.UserDefaultTotalNumberOfVisitsKey))
```

Comment:
I suggest only to use `self.` where enforced by the compiler. Less clutter and typing.
Style question, up to you.